### DISCUSSION TITLE
create detailed image report from command line

### DISCUSSION THREAD

**Discussion by McBluna** [WinMerge/winmerge#1760] [Show and tell]
Hello,
I've started this as a question under Q&A but I've solved it myself and would like to share it with you.
In GUI mode, I've the possibility to expand the folders or disable tree mode and then create and detailed HTML report containing all the comparison images.
![grafik](https://user-images.githubusercontent.com/32108408/224966445-d8bd9af8-0ca1-460a-a67e-929f370f9eb4.png)
![grafik](https://user-images.githubusercontent.com/32108408/224966994-7c1c15e3-d338-474d-aab3-4a95dcd48854.png)
![grafik](https://user-images.githubusercontent.com/32108408/224968583-7f11b530-d510-44ca-bbf5-e707cb724048.png)
![grafik](https://user-images.githubusercontent.com/32108408/224968625-aaf2004e-e3d5-4365-ad06-4aabcaefbd56.png)

I would like to do the same via command line, but I'm not able to do anything to expand the folders or disable tree mode. Is there anything I'm doing wrong?
`"C:\Program Files\WinMerge\WinMergeU.exe" /r /noninteractive /dl "2023-03-13" /dr "2023-03-14" "K:\Install\Games\Top War\Hero Sheets\2023-03-13" "K:\Install\Games\Top War\Hero Sheets\2023-03-14" /or "C:\Users\McBlu\OneDrive\Dokumente\Top War\FTP-Server\Hero Sheets\2023-03-14.html"`
![grafik](https://user-images.githubusercontent.com/32108408/224969349-7e68c2e7-f46f-46df-8e0b-dc8953ca24cf.png)

I've tried this option but it makes no difference
![grafik](https://user-images.githubusercontent.com/32108408/224972758-5611faf0-464f-4b54-abc1-b077c82fd873.png)

This did the trick
![grafik](https://user-images.githubusercontent.com/32108408/224975510-07ac0a32-12e6-4740-80ec-929ab1ba3c8f.png)

**Comment by sdottaka** [comment]
Thanks for sharing. Although not documented, the following command line option will give the expected report regardless of the Options dialog setting.

~~~
-cfg Settings/DirViewExpandSubdirs=1 # Automatically expand all subfolders
-cfg ReportFiles/ReportType=2 # Style: Simple HTML
-cfg ReportFiles/IncludeFileCmpReport=1 # Include File Compare Report
~~~
